File: testes/android-developer/Movie DB Test/app/src/main/java/com/clickbus/moviesdbtest/movies/models/Movie.kt
package com.clickbus.moviesdbtest.movies.models

import com.google.gson.annotations.SerializedName

data class Movie(
    @SerializedName("id") val id: Int,
    @SerializedName("title") val title: String,
    @SerializedName("poster_path") val posterPath: String?,
    @SerializedName("adult") val adult: Boolean,
    @SerializedName("overview") val overview: String,
    @SerializedName("release_date") val releaseDate: String,
    @SerializedName("genre_ids") val genreIds: List<Int>?,
    @SerializedName("original_title") val originalTitle: String,
    @SerializedName("original_language") val originalLanguage: String,
    @SerializedName("backdrop_path") val backdropPath: String?,
    @SerializedName("vote_count") val voteCount: Int,
    @SerializedName("video") val video: Boolean,
    @SerializedName("popularity") val popularity: Double,
    @SerializedName("vote_average") val voteAverage: Double
)

File: testes/ios-developer/MoviesDB/MoviesDB/Source/Model/Movie.swift
//
//  Movie.swift
//  MoviesDB
//
//  Created by Ruan Reis on 09/10/20.
//

import Foundation

struct Movie: Decodable {

    let id: Int

    let title: String

    let overview: String
    
    let releaseDate: String
    
    let genreIds: [Int]
    
    let voteCount: Int
    
    let voteAverage: Double
    
    let posterPath: String?
    
    let backdropPath: String?
    
    var relaseDateFormatted: String? {
        releaseDate
            .asDate(format: "yyyy-MM-dd")?
            .stringFormat
    }
}
